chapters/annotations.tex

```diff
@@ -162,23 +162,26 @@ \subsubsection{Axis Properties}\label{axis-properties}
 record Axis
   Real min "Axis lower bound";
   Real max "Axis upper bound";
-  String unit "Unit of min and max";
+  String unit = "" "Unit of min and max";
   String label "Axis label";
 end Axis;
 \end{lstlisting}
 
 When an axis bound isn't provided, the tool computes one automatically.
 
-The Modelica tool is responsible for showing the unit used for values at the
-axis tick marks, so the axis \lstinline!label! shall not contain the unit.
+An empty \lstinline!unit! means that the axis is unitless, and each expression plotted against it may use its own unit determined by the tool.  The tool is responsible for conveying the information
+about choice of unit for the different variables, for instance by attaching this information to curve legends.
 
-If a tool does not recognize the \lstinline!unit!, it is recommended to issue a
-warning and treat the \lstinline!unit! as if it was not specified.
+\begin{nonnormative}
+When \lstinline!unit! is empty, and axis bounds are to be determined automatically, a natural choice of unit could be the variable's \lstinline!displayUnit!.  When axis bounds are specified by the
+user, on the other hand, a tool may choose a unit for the variable such that the range of the variable values (expressed in the chosen unit) fit nicely with the range of the unitless axis.
+\end{nonnormative}
+
+If a tool does not recognize the \lstinline!unit!, it is recommended to issue a warning and treat the \lstinline!unit! as if it was empty.
```

Comment:
Suggestion that came out of the discussion above:
```suggestion
If a tool does not recognize the \lstinline!unit!, it is recommended to issue a warning and treat the \lstinline!unit! as if it was empty, as well as ignore any setting for \lstinline!min! and \lstinline!max!.
```

Reply:
Agreed.